src/backend/utils/compatibility_layers.ts

```diff
@@ -152,10 +152,14 @@ export async function getLinuxWineSet(
 
   const protonPaths = [`${toolsPath}/proton/`]
 
+  const { showValveProton } = GlobalConfig.get().getSettings()
+
   await getSteamLibraries().then((libs) => {
     libs.forEach((path) => {
-      protonPaths.push(`${path}/steam/steamapps/common`)
-      protonPaths.push(`${path}/steamapps/common`)
+      if (showValveProton) {
+        protonPaths.push(`${path}/steam/steamapps/common`)
+        protonPaths.push(`${path}/steamapps/common`)
+      }
```

Comment:
the reason I was doing this filtering in the frontend was to not end up with an empty wine selector

if the filter happens in the backend and a user already has Proton configured for a game, the wine selector won't have that version and it will show the empty select, making it look like there's some issue

Reply:
but if they are already using proton for a game which is already working we don't want to make them change it, right? if it's working already they'll think they have to pick another proton and maybe the game stops working (I know for example some Genshin Impact requires Proton 10 because it doesn't work with GE-Proton https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/wiki/Anime-Games#genshin-impact

Reply:
I've now changed the selector to also add the currently set version to the list if it's still valid. That should resolve the issue while still making the two options independent